freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_healthagent_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+#
+# Add anonymous access only for replication agreements
+#
+export HOSTCN=`hostname -f| sed -E 's/^|\./,dc=/g' | sed 's/^,[^,]*,//g'`
+
+ldapmodify -x -D "cn=directory manager" -w $FPW  -h $HOSTNAME << EOF
+dn: cn="$HOSTCN",cn=mapping tree,cn=config
+changetype: modify
+add: aci
+aci: (targetattr=*)(targetfilter="(|(objectclass=nsds5replicationagreement)(objectclass=nsDSWindowsReplicationAgreement))")(version 3.0; aci "permission:Read Replication Agreements"; allow (read, search, compare) groupdn = "ldap:///anyone";)
+EOF
+
+#
+# Add the cert extraction to run when the certificate is updated
+#
+ipa-getcert start-tracking -n Server-Cert -d /etc/httpd/alias -C "/usr/libexec/ipa/certmonger/restart_httpd;systemctl restart freeipa_healthagent"
```

Comment:
does it cause any issue on a system which don't have the agent installed?

Reply:
removed.  will make it part of the RPM